cluster/environments/azure-multiple-clusters/README.md

```diff
@@ -15,6 +15,52 @@ is created in.
 
 **This is a placeholder, this repo currently only deploys the second method -- where the Public IP is in the same resource group as the AKS cluster**
 
+Deploying this environment requires a service principal.  To create that service principal, one uses the Azure CLI and issues the following command:
```

Comment:
Environment can use be deployed with either user account or Service Principal authentication to Azure so it is optional.

Reply:
I think this should be covered in a general section, so I'll create an issue specific to SPs and login.  See issue #198